Mentioned json schema for tsconfig.json (#69)
TypeScriptToLua/TypeScriptToLua.github.io: docs/configuration.md

To customize transpilation behavior we add a new group of options to the `tsconfig.json` file. All of these options should be placed in a `tstl` object.

> Optionally, you can specify the json schema in the `$schema` property to recieve custom options hints from the IDE.

Reviewer: ```suggestion
You can use our [VS Code extension](editor-support.md) or manually specify the json schema to receive autocompletion and hints for the configuration file.
```

> You can use our [VS Code extension](editor-support.md) or manually specify the json schema to receive autocompletion and hints for the configuration file.

Reviewer: ```suggestion
You can use our [VS Code extension](editor-support.md) or manually specify the JSON schema to receive autocompletion and hints for the configuration file.
```

```json title=tsconfig.json
{

>   // Schema file for hints and validation

Reviewer: ```suggestion
  // Optional: Schema file for hints and validation
```
